templates/web/highwaysengland/report/form/user.html

```diff
@@ -1,6 +1,23 @@
 <!-- report/form/user.html -->
-<div class="js-new-report-user-hidden form-section-preview form-section-preview--next">
-    <button name="report_anonymously" value="yes" class="btn btn--block js-new-report-submit">Report anonymously</button>
-    <small>No personal details will be stored.</small>
+<div class="form-section-preview form-section-preview--next hidden-nojs">
+    <h2 class="form-section-heading form-section-heading--private hidden-nojs">
+        [% loc('Next:') %] [% loc('Tell us about you') %]
+    </h2>
+
+<button type="button" class="btn btn--block hidden-nojs js-reporting-page--next">Continue and provide contact details</button>
```

Comment:
Clicking this button works fine for creating reports, but doesn't seem to work when trying to post an update.

I think it's related to the code below. The update form doesn't have the `js-reporting-page` class, so the handler isn't being triggered on the updates page.

https://github.com/mysociety/fixmystreet/blob/2762bd7bbfe8233011d4e9174932d69f2d25fade/web/cobrands/fixmystreet/fixmystreet.js#L1032-L1042

Reply:
Oh, for some reason I thought there weren't updates on the NH site so didn't check it properly/didn't make sure it had the right bits in. But clearly there are! Will work it out.